.travis.yml

```diff
@@ -27,31 +22,20 @@ jobs:
 
     - name: "Generate report.hml"
       install:
-        - sudo apt-get install gperf -y
-        - sudo apt-get install libfl-dev -y
-        - sudo apt-get install npm
-        - sudo apt-get install gcc-8 -y
-        - sudo apt-get install g++-8 -y
-        - sudo apt-get install yosys -y
-        - sudo apt-get install iverilog -y
-        - sudo apt-get install verilator -y
-        - sudo ln -sf /usr/bin/gcc-8 /usr/bin/gcc
-        - sudo ln -sf /usr/bin/g++-8 /usr/bin/g++
-        - sudo ln -sf /usr/bin/ccache /usr/local/bin/gcc
-        - sudo ln -sf /usr/bin/ccache /usr/local/bin/g++
-        - sudo ln -sf /usr/bin/ccache /usr/local/bin/gcc-8
-        - sudo ln -sf /usr/bin/ccache /usr/local/bin/g++-8
+        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+        - bash miniconda.sh -b -p $HOME/miniconda
+        - source "$HOME/miniconda/etc/profile.d/conda.sh"
+        - bash -r
+        - conda config --set always_yes yes --set changeps1 no
+        - conda update -q conda
+        - conda info -a
+        - conda config --add channels symbiflow
+        - conda create -q -n travis verilator yosys iverilog
+        - conda activate travis
         - pip install -r conf/requirements.txt
-        - mkdir -p ~/.local/bin
-        - export PATH=$HOME/.local/bin:$PATH
-        - travis_retry curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
       script:
-        - "make slang"
-        - "make odin"
-        - "make zachjs-sv2v"
-        - "make tree-sitter-verilog"
         - "make generate-tests"
-        - "make report USE_ALL_RUNNERS=1"
```

Comment:
Keep this. If something is wrong with one of the packages this will detect it.

Reply:
I intend to keep this but for the moment I wanted to check how it works only with tools currently available from anaconda and with this it would fail at startup

Reply:
:+1: